validation/src/main/scala/hmda/validation/rules/lar/quality/Q004.scala

```diff
@@ -0,0 +1,23 @@
+package hmda.validation.rules.lar.quality
+
+import com.typesafe.config.ConfigFactory
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.dsl.Result
+import hmda.validation.rules.EditCheck
+import hmda.validation.dsl.PredicateCommon._
+import hmda.validation.dsl.PredicateSyntax._
+
+object Q004 extends EditCheck[LoanApplicationRegister] {
+  override def apply(lar: LoanApplicationRegister): Result = {
+
+    val config = ConfigFactory.load()
+    val loanAmount = config.getInt("hmda.validation.quality.Q004.loan.amount")
+
+    when((lar.loan.loanType is equalTo(3) and
+      (lar.loan.propertyType is containedIn(List(1, 2))))) {
```

Comment:
My IDE thinks there are unnecessary parens here, and I tend to agree with it.
Also, same comment about a weird git history as elsewhere.  (`git log --follow` to see what I mean.)